docs/installation_guide/upgrades/0_10_0.rst

```diff
@@ -0,0 +1,85 @@
+==============
+Version 0.10.0
+==============
+
+Version 0.10.0 requires some changes to the cluster definition. 
+
+Changes to the cluster definition
+---------------------------------
+
+1. In the global `attrs` section add the current version attribute. The attribute should be set to your current installed version. Example: 
+
+.. code-block:: yaml
+
+    attrs:                                                                                                         
+        install:                                                                                                     
+            current_version: "0.9.1"
+
+2. Hopsworks 0.10.0 drops support for Zeppelin. As such, the Zeppelin cookbook is not maintained anymore. Please remove the `- zeppelin` entry from your cluster definition
+
+3. Hopsworks 0.10 add support for PyTorch. PyTorch is installed in the base environments and it will be available in all the newly created environments after the upgrade. PyTorch is installed from the Conda repository. If you are using an internal mirror you need to make the `PyTorch <https://anaconda.org/pytorch/repo>`_ channel available. Alternatively you can rely on the public mirror, but be careful about rate limiting! The attributes to specify the Conda internal mirrors have changed to:
+
+.. code-block:: yaml
+
+    attrs:                                                                                                         
+        conda:
+            channels:
+                default_mirrors: "[main/free mirror location]"
+                pytorch: "[pytorch mirror location]"
+            use_defaults: "false"
+
+4. Hopsworks 0.10 makes available to the users more logging information related to operations to the python environments. This is achieved using filebeat. In the cluster definition, on all the node groups, you should add `      - hopslog::_filebeat-kagent` in the recipe list.
+
+5. Remember to add the `hopsworks::migrate` recipe in the same node group which the Hopsworks webapp belongs to. This will run `Expat` which will migrate old job configurations to the new format and create the Kibana index pattern for the Python environment logs. Migrate will also change the storage type of the `/Project` directory on HopsFS to use, whenever possible, NVME disks as storage.
```

Comment:
@SirOibaf `webapp` is a term readers of this document are not familiar with. We can instead say they need to add the recipe in the same group as the `hopsworks` recipe.